curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md

```diff
@@ -8,12 +8,11 @@ dashedName: step-1
 # --description--
 
 HTML elements have opening tags like `<h1>` and closing tags like `</h1>`.
+The text for an element goes between its opening and closing tags.
```

Comment:
If you want to have a new line here, you need to have one empty line between the paragraphs

Reply:
That does look better. Ill commit with changes